### ISSUE TITLE
Is there a seperation function in eatGADS to seperate strings into multiple columns?

### ISSUE THREAD

**Issue by nickhaf** [beckerbenj/eatRecode#19]

**Comment by beckerbenj** [comment]
No, unfortunately no respective function exists in `eatGADS`. I have created a development branch in `eatGADS`, set up a `splitGADS()` function and outlined its possible structure. Unfortunately, I do not have the time to implement and test the function.

Could you @nickhaf take over this issue? PF should be able to review.
